### ISSUE TITLE
No story title in exported sketch [BUG]

### ISSUE THREAD

**Issue by SecSamDev** [google/timesketch#1300]
The titles of the stories are not being saved correctly when exporting the sketch.
```
Unknown Title
Story Metadata:

    Creation Date:
    Created by: N/A
    Report Generated: 2020-07-10T11:59:34.954132
    Report Exported by: N/A
```
```html
<h1>Unknown Title</h1>

Story Metadata:<br />
<ul>
    <li><b>Creation Date:</b> </li>
    <li><b>Created by:</b> N/A</li>
    <li><b>Report Generated:</b> 2020-07-10T11:59:34.954132</li>
    <li><b>Report Exported by:</b> N/A</li>
</ul>
```

**Comment by kiddinn** [comment]
the story had a title, I assume?

let me take a look at this.

**Comment by kiddinn** [comment]
the exporter didn't set the title as well as few other settings, fixed in #1321

**Comment by SecSamDev** [comment]
Thank you
